source/documentation/deploying_services/index.md

```diff
@@ -1008,6 +1005,102 @@ You must bind your app to the Redis service to be able to access the cache from
 
     If your app writes service connection errors to `STDOUT` or `STDERR`, you can view recent errors with ``cf logs APPNAME --recent``. See the section on [Logs](#logs) for details.
 
+### Connect to a Redis service instance from your local machine
+
+You must use the [Conduit](#conduit) plugin and stunnel tool to connect your local machine to your Redis service instance.
```

Comment:
Is it worth linking to the section on installing `stunnel`?

Reply:
The word "must" here seems a bit strong. There's no requirement to use conduit to achieve this. It's probably the easiest way to do it, but there are other approaches such as manually creating ssh tunnels.

I also wonder whether we need to call out why stunnel is necessary (ie the redis-cli doesn't have TLS support). If using a different client that does have TLS support, stunnel wouldn't be necessary.